packages/jaeger-ui/vite.config.ts

```diff
@@ -64,33 +64,6 @@ export default defineConfig({
       less: {
         math: 'always',
         javascriptEnabled: true,
-        modifyVars: {
-          // Supply appropriate overrides to the Ant Design System.
```

Comment:
Can you explain what's happening here?

Reply:
Here, we are basically removing the replacement variables.

Before v5, we had an option to replace less variables in ant-design's core files with the ones defined here.
This `modifyVars` object was containing all such variables. But since v5, antd doesn't support less variables, so, this modification thing is of no use.

Instead, ant-design introduced seedTokens and aliasTokens which can be supplied to the theme directly.

Reply:
is there documentation about that? The names seedToken don't tell me much, especially why they would be a replacement for less variables.

On a high level, what is the approach now? My understanding was that less variables allowed us to reuse settings across different css files. Do we have to replicate those now in many places?

Reply:
[ant-design v4 to v5 migration guide](https://ant.design/docs/react/migration-v5#less-migration:~:text=Remove%20less%2C%20adopt,antd%20specification%20style.) explaining why they removed the support for less variables.

[This document](https://ant.design/docs/react/customize-theme#life-of-design-token) explains in deep how to migrate from less variables to tokens and what exactly is all that seedToken, and other such token thing.

Since we were replacing those variables via the `modifyVars` object, the same can be (is achieved) using the design tokens supplied to the theme configuration of ant-design.

less variables for sure achieves the purpose of reusing them across files, but since the change is internal to ant-design library, we aren't affected in other areas, as we aren't using any antd's less file in our code.